ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -86,6 +86,8 @@ param set MAV_SYS_ID $((px4_instance+1))
 simulator_tcp_port=$((4560+px4_instance))
 udp_offboard_port_local=$((14580+px4_instance))
 udp_offboard_port_remote=$((14540+px4_instance))
+udp_onboard_payload_port_local=$((14558+px4_instance))
```

Comment:
@Jaeyoung-Lim What pattern do you recommend here?

Reply:
This will conflict with `udp_offboard_port_local` after 22 instances. Would it be possible to find another slot? Regarding patterns I think the first port is defined up to the decimal, to be consistent with the others

Reply:
which port do you suggest?

Reply:
The used ports are only from `18570 +0~255` and `14580+0~255` so anything outside of this...so maybe `14280` would do the trick? Just a question, what is the downside of using the `udp_offboard_port_local` port for the onboard payload? This will only be used for development (SITL) and I cannot imagine too many people trying to use both at the sametime

Reply:
The camera is a different component than actual off board == mission management interfaces. You can't mix them.